source/codegen/metadata/nifake/CHANGES.md

```diff
@@ -0,0 +1,16 @@
+# Changes in metadata from the nimi-python metadata
+
+## functions.py
+
+The following functions, not originally in nimi-python metadata, were newly added:
+- `'InitExtCal'`
+- `'CloseExtCal'`
+ 
+The following functions were tagged with `'init_method': True,` to ensure their generated service handlers register the new session
+with the session_repository.
+- `InitWithOptions`
```

Comment:
@Akshata440 , @doshirohan , do you know why Changes.md file was not present for niFake before? This change of adding init_method tag to InitWithOptions should have been documented early itself know?

Reply:
I am not sure. I have added it in dcpower branch for new function I had introduced for ViBoolean[] parameter.

Reply:
Yeah maybe there werent any changes made to the metadata I guess. I think @reckenro would know.

Reply:
This is niFake meta data location rigth? https://github.com/ni/nimi-python/blob/master/src/nifake/metadata/functions.py 
I could not fine init_method there